src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -312,6 +320,13 @@ class ProfileCallTreeContextMenu extends PureComponent<Props> {
             ? 'Focus on calls made by this function'
             : 'Focus on function'}
         </MenuItem>
+        <MenuItem
+          onClick={this.handleClick}
+          data={{ type: 'collapse-function-subtree' }}
+        >
+          <span className="profileCallTreeContextMenuIcon profileCallTreeContextMenuIconCollapse" />
+          {"Collapse function's subtree"}
```

Comment:
Actually I think we need to make it clearer that this happens accross the whole tree => `Collapse this function’s subnodes accross the tree`
I changed "subtree" so that I can use the word "tree" without a repetition. You could also use "descendants" but I wasn't sure how accurate this term would be.

Reply:
I have no way to type the nice quote directly and have never used it in my English writing, not sure why we should make it more complicated than it needs to be.

Reply:
I updated the label to be more explicit.